apidoc/Titanium/UI/WebView.yml

```diff
@@ -1121,6 +1152,41 @@ properties:
     since: {iphone: "8.0.0", ipad: "8.0.0", macos: "9.2.0"}
     permission: read-only
 
+  - name: DIN_A5
+    summary: PDF paper size
+    type: Number
+    platforms: [android]
+    permission: read-only
+    since: "12.1.0"
+
+  - name: DIN_A4
+    summary: PDF paper size
+    type: Number
+    platforms: [android]
+    permission: read-only
+    since: "12.1.0"
+
+  - name: DIN_A3
+    summary: PDF paper size
+    type: Number
+    platforms: [android]
+    permission: read-only
+    since: "12.1.0"
+
+  - name: DIN_A2
+    summary: PDF paper size
+    type: Number
+    platforms: [android]
+    permission: read-only
+    since: "12.1.0"
+
+  - name: DIN_A1
```

Comment:
Can you change to `PDF_PAGE_DIN_A2`, etc.? It makes it more obvious where it's used.

Reply:
done :+1: